edi_storage_oca/readme/CONTRIBUTORS.rst

```diff
@@ -1 +1,2 @@
 * Simone Orsi <simahawk@gmail.com>
+* Foram Shah <foram.shah@initos.com>
```

Comment:
@fshah-initos you can add yourself if here if you provide something more than a bare migration.
TIP: test coverage is very welcomed :wink: